vagrant_config.json

```diff
@@ -208,7 +208,8 @@
       },
       "ruby":{
         "enabled":true,
-        "version": "1.9.3-p194",
+        "versions": ["1.9.3-p551", "2.2.3"],
+        "global_version": "1.9.3-p551",
```

Comment:
and need to check, update docs for this, too: http://dev.teracy.org/docs/develop/ruby_dev_guide.html